### PR TITLE
chore: add -ct for future component testing

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export function executeTemplate(input: string, args: Record<string, string>, sec
   let mode: 'show' | 'hide' | 'comment' = 'show';
   let indent = '';
   for (const line of input.split('\n')) {
-    const match = line.match(/([\s]+)\/\/--begin-(.*)/);
+    const match = line.match(/([\s]*)\/\/--begin-(.*)/);
```
Comment (rwoll) [inline src/utils.ts:73]:
```suggestion
    const match = line.match(/(\s*)\/\/--begin-(.*)/);
```

File: src/generator.ts [hunk 2]
```diff
@@ -175,11 +224,25 @@ export class Generator {
...
   }
 
+  private async _patchTsconfigJSON() {
+    const tsconfigFile = path.join(this.rootDir, 'tsconfig.json');
+    const files = new Map<string, string>();
+    if (!fs.existsSync(tsconfigFile)) {
+      files.set(`tsconfig.json`, this._readAsset(path.join('tsconfig.json')));
+    } else {
+      const tsconfigJSON = fs.readFileSync(path.join(this.rootDir, 'tsconfig.json'), 'utf-8');
+      const newJSON = tsconfigJSON.replace(/("include"[\s\S]*:[\s\S]\[[\s\S]*"src")/m, '$1, "playwright/playwright.d.ts"');
```
Comment (rwoll) [inline src/generator.ts:234]: Looks like this assumes they have `"src"` in their `include` already? Why is that a legit assumption?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-playwright",
-  "version": "1.17.110",
+  "version": "1.17.111",
```
Comment (rwoll) [inline package.json:3]: What's our version strategy here? Should we match playwright-core for at least the major version?

**Review by rwoll — COMMENTED** [review]
Roughly looks ok (especially for being authored from memory!), but let's make sure we test 😄

File: src/generator.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/generator.ts]: I'd say it is Okay for now...

File: package.json (hunk 3 above)
Comment (pavelfeldman) [reply package.json:3]: Nope, in fact this should have been 0.0.111, but someone decided to push it as semver.

**Comment by pavelfeldman** [comment]
Too early to test, it is likely all to change!

**Pull request merged**